styles/components/_tooltip.scss

```diff
@@ -4,9 +4,32 @@
 .sprk-c-Tooltip__container {
   position: relative;
   display: inline-flex;
+}
+
+/// Vertically aligns the tooltip to the top.
+/// @name .sprk-c-Tooltip__container--top
+.sprk-c-Tooltip__container--top {
+  vertical-align: top;
+}
```

Comment:
Looks great to me!

Did we look into using our existing utility classes - `sprk-u-VerticalAlign--bottom` etc? I know those classes apply `!important` and also we don't have `additionalClasses` on the tooltip container, so maybe that would not have worked.